src/siema.js

```diff
@@ -533,9 +533,15 @@ export default class Siema {
       // if dragged element is a link
       // mark preventClick prop as a true
       // to detemine about browser redirection later on
-      if (e.target.nodeName === 'A') {
-        this.drag.preventClick = true;
-      }
+
+      let target = e.target
+      do {
+        if (target.nodeName === 'A') {
```

Comment:
I've suggest to use `nodeName.toLowerCase()` and compare with `a`. You never know what case will be returns (IE eg)

Reply:
Good point :) I promise to come here and merge all relevant PRs. One day. To be honest I do not care about Siema that much anymore...

https://twitter.com/pawelgrzybek/status/1045199798838669312

Reply:
Maybe I can be a maintainer? SIEMA is so awesome for me 😄

Reply:
Can we do a Skype chat one day or catch up in Warsaw about it? Short answer - yes :)

Reply:
Yes!

Reply:
UX issues aside, was there an update on this? Unfortunately, some of us still end up having to build carousels!

Reply:
How can i edit siema.js from src, and be able to use it?

The unminified siema doesnt work, only min.js

So i cant use your changes, and i really need to try that

I want to compile a new siema with your modification, im using v1.51 but have the link-open-bug when sliding link content